Content.Shared/Item/SharedItemSystem.cs

```diff
@@ -138,6 +142,34 @@ private void AddPickupVerb(EntityUid uid, ItemComponent component, GetVerbsEvent
         args.Verbs.Add(verb);
     }
 
+    // SS220 mouth container begin
+    /// <summary>
+    ///     Transmits item on get alternative verb to MouthContainerSystem.
+    /// </summary>
+    private void OnGetAltVerb(Entity<ItemComponent> ent, ref GetVerbsEvent<AlternativeVerb> args)
+    {
+        var user = args.User;
+
+        if (TryComp<MouthContainerComponent>(user, out var mouthComp))
+        {
+            var toInsert = ent.Owner;
+            if (_mouthSystem.CanInsert((user, mouthComp), toInsert))
+            {
+                var v = new AlternativeVerb
+                {
+                    Priority = 1,
+                    Text = Loc.GetString(mouthComp.InsertVerbIn),
+                    Disabled = false,
+                    Impact = LogImpact.Medium,
+                    DoContactInteraction = true,
+                    Act = () => { _mouthSystem.TryInsert((user, mouthComp), user, toInsert); },
```

Comment:
Тут в блок, по идее, можно не выделять
```suggestion
                    Act = () => _mouthSystem.TryInsert((user, mouthComp), user, toInsert),
```